### ISSUE TITLE
The loss cannot be reduced

### ISSUE THREAD

**Issue by yilonghe** [sming256/OpenTAD#37]
For the Multi-THUMOS dataset, I can obtain the expected results using the provided weights for testing. However, when training from scratch, after the third epoch, the loss fluctuates around 1.1 and does not decrease further. There have been no changes to the code or commands. Could you provide some possible reasons for this issue?

**Comment by sming256** [comment]
Please make sure the following are correctly set.

- THUMOS raw video data and Multi-THUMOS annotations: please use the provided data.
- Pretrained VideoMAE checkpoint: AdaTAD is initialized with K400-pretrained VideoMAE; please make sure you download them and put them under the folder ./pretrained/ correctly, as illustrated [here](https://github.com/sming256/OpenTAD/tree/main/configs/adatad#prepare-the-pretrained-videomae-checkpoints).

If you have any further questions, please email me.
